Source/OpalGraphics/cairo/CairoFontX11.m

```diff
@@ -163,22 +169,23 @@ - (CFStringRef) copyFullName;
         result = [[NSString alloc] initWithBytes: nameStruct.string length: nameStruct.string_len encoding: NSUTF16BigEndianStringEncoding];
       }
       else if (nameStruct.platform_id == TT_PLATFORM_MACINTOSH &&
-               nameStruct.encoding_id == TT_MAC_ID_ROMAN)
+        nameStruct.encoding_id == TT_MAC_ID_ROMAN)
       {
         result = [[NSString alloc] initWithBytes: nameStruct.string length: nameStruct.string_len encoding: NSMacOSRomanStringEncoding];
       }
       else if (nameStruct.platform_id == TT_PLATFORM_MICROSOFT &&
-               nameStruct.encoding_id == TT_MS_ID_UNICODE_CS)
+        nameStruct.encoding_id == TT_MS_ID_UNICODE_CS)

```

Comment:
I'd agree with @ivucica : for multiline expressions the lines after the first one shouldn't be indented left of the opening paren.

Here's an example in libs-gui:
https://github.com/gnustep/libs-gui/blob/master/Source/NSTextView.m#L1677